android/build.gradle

```diff
@@ -1,33 +1,52 @@
-apply plugin: 'com.android.library'
+buildscript {
+    ext.safeExtGet = {prop, fallback ->
+        rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+    }
+    repositories {
```

Comment:
Please also consider adding an if statement to conditionally loads Android Gradle Plugin. you can refer to [this PR](https://github.com/facebook/react-native/pull/25569) especially [this comment](https://github.com/facebook/react-native/pull/25569#issuecomment-532504277) for finding the reason why it's needed.

To do that just refer to [this PR on react-native-svg](https://github.com/react-native-community/react-native-svg/pull/1156/files) and apply similar changes on this repository too.

Reply:
Thanks for the feedback, I applied this change. Please check it out and resolve if it's OK.

Reply:
Thanks for your work, but one more thing,
there is no longer need to import Android Gradle Plugin version from the root project,  please change it to 

```
classpath("com.android.tools.build:gradle:3.5.3")
```

and also for having a change similar to the PRs on other repository, consider to add the comments above the `if` statement (just take a look at the react-native-svg's PR and copy and paste it in the build.gradle file.